spec/controllers/account_reset/cancel_controller_spec.rb

```diff
@@ -4,9 +4,6 @@
   include AccountResetHelper
 
   let(:user) { create(:user, :signed_up) }
-  before do
-    TwilioService::Utils.telephony_service = FakeSms
```

Comment:
If this is the new default, then it might make sense to move this to config.before(:each)

Reply:
Not sure I understand. `FakeSms` is gone and was replaced by `FakeRestClient`. I set that in a before block for the suite here: https://github.com/18F/identity-idp/pull/3005/files#diff-e11bdaf9116be3911e9557b69e242370R69